package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "prepublishOnly": "rimraf lib && babel src -d lib",
     "start": "rimraf lib && babel src -d lib -w",
-    "test": "jest"
+    "test": "babel src -d lib && jest"
```

Comment:
Could you call `npm run prepublishOnly` instead? To avoid duplication of the build script.